.github/workflows/build.yml

```diff
@@ -40,3 +44,35 @@ jobs:
 
     - name: Test
       run: ./bin/phpunit
+
+  publish_docker_images:
+    needs: build
+    runs-on: ubuntu-latest
+
+    if: github.ref == 'refs/heads/master' || github.event_name == 'release'
```

Comment:
@micheleorselli we need to filter actions only for: pushes on master and new tags.